rpc/core/types/responses.go

```diff
@@ -12,6 +13,18 @@ import (
 	"github.com/tendermint/tendermint/types"
 )
 
+// List of standardized errors used across RPC
+var (
+	ErrZeroOrNegativePerPage  = errors.New("zero or negative per_page")
+	ErrPageOutOfRange         = errors.New("page should be within range")
+	ErrZeroOrNegativeHeight   = errors.New("height must be greater than zero")
+	ErrHeightExceedsChainHead = errors.New("height must be less than or equal to the head of the node's blockchain")
+	ErrHeightNotAvailable     = errors.New("height is not available")
+	// ErrInvalidRequest is used as a wrapper to cover more specific cases where the user has
+	// made an invalid request
+	ErrInvalidRequest = errors.New("invalid request")
+)
```

Comment:
Are these being used anywhere?

Reply:
They do not appear to be, good catch. I have removed them from the patch.